docs/site/index/downloads.md

```diff
@@ -196,3 +196,500 @@ The 1.0 version of CLDR is described here for historical interest only. It was h
 | 1.0 DTDs: | https://www.openi18n.org/spec/ldml/1.0/ldml.dtd |
 |  | https://www.openi18n.org/spec/ldml/1.0/ldmlSupplemental.dtd |
 
+[CLDR-18087]: https://unicode-org.atlassian.net/browse/CLDR-18087
+
+<!-- Latest release -->
+[latest-version]: /index/downloads/latest
+[latest-data]: https://unicode.org/Public/cldr/46/
+[latest-charts]: https://www.unicode.org/cldr/charts/46/index.html
+[latest-ldml]: https://www.unicode.org/reports/tr35/
+[latest-tickets]: https://unicode-org.atlassian.net/issues/?jql=project%20%3D%20CLDR%20AND%20status%20%3D%20Done%20AND%20resolution%20%3D%20Fixed%20AND%20fixVersion%20%3D%20%2246%22%20ORDER%20BY%20created%20ASC
+[latest-delta-dtd]: https://www.unicode.org/cldr/charts/46/supplemental/dtd_deltas.html)
+
+<!-- Dev release -->
+[dev-version]: /index/downloads/dev)
```

Comment:
I noticed a few other cases where there was an extra trailing ")". Probably want to do a search on the page, then check the highlighted ones after line 200ish.